pwndbg/commands/context.py

```diff
@@ -82,14 +83,22 @@ def output():
         return open(str( config_output ), "w")
 
 # @pwndbg.events.stop
```

Comment:
Can we remove this comment?

Reply:
Looks like this was added a long time ago. Should be fine to remove.